client/src/components/WorkflowInvocationState/WorkflowInvocationSummary.vue

```diff
@@ -126,6 +136,28 @@ export default {
         invocationState: function () {
             return this.invocation?.state || "new";
         },
+        invocationStateSuccess: function () {
+            return this.invocationState == "scheduled" && this.runningCount === 0 && this.invocationAndJobTerminal;
```

Comment:
```suggestion
            return this.invocationAndJobTerminal;
```
so you might as well drop this